sdk/lib/atl/cstringt.h

```diff
@@ -858,7 +858,7 @@ class CStringT :
         PXSTR pszResult = CThisSimpleString::GetBuffer(nResultLength);
         PXSTR pszNext;
         int nCount = 0, nRemaining = nLength;
-        while (nRemaining && (pszNext = StringTraits::FindString(pszResult, pszOld)))
+        while (nRemaining && (pszNext = const_cast<PXSTR>(StringTraits::FindString(pszResult, pszOld))))
```

Comment:
To avoid the need for this, you could have two overloads of FindString -- much like wcsstr/strstr: one with const input and const return and one with non-const input and non-const return.

Reply:
Fixed. Thanks!

Reply:
Now we could fix this issue without using `const_cast`.